### PR TITLE
doc(Samples): Add readme for Pnp Service samples

### PR DESCRIPTION
**Pull request by bikamani**

### CODE REVIEW

File: iothub/service/samples/readme.md [hunk 1]
```diff
@@ -1,14 +1,85 @@
...
+
+* [ADM Sample][adm-sample]
+* [Jobs Sample][jobs-sample]
+* [Registry ManagerSample][reg-man-sample]
+* [Service Client Sample][service-client-sample]
+* [Plug and play Samples ][pnp-service-sample]
+
+### Prerequisites
+In order to run the device samples on Linux or Windows, you will first need the following prerequisites:
+* [Setup your IoT hub][lnk-setup-iot-hub]
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:15]: `[lnk-setup-iot-hub]` is missing

File: iothub/service/samples/readme.md [hunk 2]
```diff
@@ -1,14 +1,85 @@
...
-* [ServiceClientSample][service-client-sample]
+Here, you'll find simple samples showing how to use the various features of the Microsoft Azure IoT Hub service from a back-end application running C# code.
+
+### List of Samples
+
+* [ADM Sample][adm-sample]
+* [Jobs Sample][jobs-sample]
+* [Registry ManagerSample][reg-man-sample]
+* [Service Client Sample][service-client-sample]
+* [Plug and play Samples ][pnp-service-sample]
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:11]: Add a link to other service SDK samples present here: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service
You can leave out the `CleanUpDevicesSample`.

File: iothub/service/samples/readme.md [hunk 3]
```diff
@@ -1,14 +1,85 @@
-Service samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-repo] repository
+## Service samples for Azure IoT Service SDK for C#
 
-### [Service samples][service-samples]
-* [ADMSample][adm-sample]
-* [JobsSample][jobs-sample]
-* [RegistryManagerSample][reg-man-sample]
-* [ServiceClientSample][service-client-sample]
+Here, you'll find simple samples showing how to use the various features of the Microsoft Azure IoT Hub service from a back-end application running C# code.
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:3]: ...features of ~the~ Microsoft Azure IoT Hub service...

File: iothub/service/samples/readme.md [hunk 4]
```diff
@@ -1,14 +1,85 @@
...
+* [ADM Sample][adm-sample]
+* [Jobs Sample][jobs-sample]
+* [Registry ManagerSample][reg-man-sample]
+* [Service Client Sample][service-client-sample]
+* [Plug and play Samples ][pnp-service-sample]
+
+### Prerequisites
+In order to run the device samples on Linux or Windows, you will first need the following prerequisites:
+* [Setup your IoT hub][lnk-setup-iot-hub]
+* After creating the IoT Hub, retreive the iothubowner connection string (mentioned in instructions to create the hub).
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:16]: typo: retrieve

Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:16]: If available, add a link to that part of the instructions: ...mentioned in instructions to create the hub

File: iothub/service/samples/readme.md [hunk 5]
```diff
@@ -1,14 +1,85 @@
...
+
+### Setup environment
+.NET Core SDK 3.0.0 or greater on your development machine. You can download the .NET Core SDK for multiple platforms from .NET. You can verify the current version of C# on your development machine using 'dotnet --version'.
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:19]: We support .NET Standard 2.0, so ideally, you can run it with a .NET Core 2.0 application as well

File: iothub/service/samples/readme.md [hunk 6]
```diff
@@ -1,14 +1,85 @@
...
+
+Note: The samples can be compiled using the NET Core SDK 2.1 SDK if the language version of projects using C# 8 features are changed to preview.
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:21]: language version is changed to preview? is there an example for this?

File: iothub/service/samples/readme.md [hunk 7]
```diff
@@ -1,14 +1,85 @@
...
+
+### Get and run the samples
+You need to first clone the repository or download the sample project folder on your machine.
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:24]: ...download on your machine - doesn't sound right. Should it be "download to your machine"?

File: iothub/service/samples/readme.md [hunk 8]
```diff
@@ -1,14 +1,85 @@
...
+
+### Plug and Play Samples
+
+Shows how to get the plug and play model ID, update properties on the twin and invoke commands.
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:28]: ...plug and play model ID for a device...
...invoke commands on a device.

File: iothub/service/samples/readme.md [hunk 9]
```diff
@@ -1,14 +1,85 @@
...
+
+#### Build and run the Pnp(plug and play) Service Sample application for a device with no components:
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:30]: We don't need to specify "pnp" here again, it is under "pnp samples" section

File: iothub/service/samples/readme.md [hunk 10]
```diff
@@ -1,14 +1,85 @@
...
+This sample uses the following simple model which has no components - [Thermostat](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json)
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:31]: This sample uses the following simple model, which has no components

File: iothub/service/samples/readme.md [hunk 11]
```diff
@@ -1,14 +1,85 @@
...
+1. Preparing the Pnp service Sample application:
+   1. Set the following environment variables on the terminal from which you want to run the application.
+
+      * IOTHUB_CONNECTION_STRING
+      * DEVICE_ID
+
+2. Building the Pnp Service Sample application:
+
+    To build the Pnp Service Sample application using dotnet, at a command prompt navigate to the **\service\samples\PnpServiceSamples\Thermostat** folder. Then execute the following command and check for build errors:
```
Comment (abhipsaMisra) [inline iothub/service/samples/readme.md:40]: ...from command prompt, navigate to "location" ...

Comment (abhipsaMisra) [reply iothub/service/samples/readme.md:40]: also, step 1 calls it "terminal", step 2 calls it "command prompt"; they need to be the same.

File: iothub/service/samples/readme.md (hunk 2 above)
Comment (bikamani) [reply iothub/service/samples/readme.md:11]: sure

File: iothub/service/samples/readme.md (hunk 9 above)
Comment (bikamani) [reply iothub/service/samples/readme.md:30]: I would prefer to keep it . There is one more title down below as well.

File: iothub/service/samples/readme.md (hunk 7 above)
Comment (bikamani) [reply iothub/service/samples/readme.md:25]: both sound right to me

**Pull request merged**